Examples/ObjectiveC/ClusteringWithImagesExample.m

```diff
@@ -0,0 +1,69 @@
+//
+//  ClusteringWithImagesExample.m
+//  Examples
+//
+//  Created by Joshua Erb on 7/30/18.
+//  Copyright © 2018 Mapbox. All rights reserved.
+//
+
+#import "ClusteringWithImagesExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleClusteringWithImages = @"ClusteringWithImagesExample";
+
+@interface ClusteringWithImagesExample () <MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+@property (nonatomic) UIImage *icon;
+
+@end
+
+@implementation ClusteringWithImagesExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Initialize the Map
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds
+                                            styleURL:[MGLStyle lightStyleURL]];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    self.mapView.delegate = self;
+    [self.view addSubview:self.mapView];
+    
+    self.icon = [UIImage imageNamed:@"circle"];
```

Comment:
Can you move this line to the `didFinishLoadingStyle` delegate method? Since it isn't it used until then, I'm not sure if setting it all up in the `viewDidLoad` is necessary.

Another option would be to hard-code these values, then you wouldn't need to include the `icon` property on the ViewController.

Reply:
Good point 👍. I think I like the idea of hardcoding.